plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiConfig.java

```diff
@@ -90,36 +91,6 @@ public boolean getUseParquetColumnNames()
         return this.shouldUseParquetColumnNames;
     }
 
-    @Config("hudi.min-partition-batch-size")
-    @ConfigDescription("Minimum number of partitions returned in a single batch.")
```

Comment:
Generally the unused configurations are marked as defunct to avoid breaking Trino while doing an upgrade

https://github.com/trinodb/trino/blob/dc7a9fd9605cb56d69e16c461a8e8d1ab0645a85/plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java#L49-L62

Reply:
Minor clarification: removal of configs still breaks Trino if the user is using them (we fail on unused/unknown configs), but the user gets an explicit error message saying the config is defunct (rather than just unknown).

Reply:
noted.